src/stylesheets/components/Auth/_button-submit.scss

```diff
@@ -0,0 +1,15 @@
+.button-submit {
+  $self: &;
+
+  &__btn--white {
+    @include neuzeit-font-semi-bold;
+    align-items: center;
+    background: $secondary-color;
+    border-radius: 10px;
+    color: #15151a;
+    font-size: 17px;
```

Comment:
usually, we prefer to set sizing in `rem` or `em` instead of `px` for supporting multiple devices with multiple base size of the font.